cmd/parca-agent/main.go

```diff
@@ -214,6 +219,10 @@ type FlagsDWARFUnwinding struct {
 	Mixed   bool `default:"true"                                    help:"Unwind using .eh_frame information and frame pointers"`
 }
 
+type FlagsTelemetry struct {
+	DisablePanicTelemetry bool `default:"false"`
```

Comment:
Maybe just `DisablePanic` or `DisableTrackingPanics` since we already have telemetry prefix?